core/commands/add.go

```diff
@@ -207,6 +218,24 @@ See 'dag export' and 'dag import' for more information.
 		inlineLimit, _ := req.Options[inlineLimitOptionName].(int)
 		toFilesStr, toFilesSet := req.Options[toFilesOptionName].(string)
 
+		if chunker == "" {
+			chunker = cfg.Import.UnixFSChunker.WithDefault(config.DefaultUnixFSChunker)
+		}
+
+		if hashFunStr == "" {
+			hashFunStr = cfg.Import.HashFunction.WithDefault(config.DefaultHashFunction)
+		}
+
+		if !cidVerSet && !cfg.Import.CidVersion.IsDefault() {
+			cidVerSet = true
+			cidVer = int(cfg.Import.CidVersion.WithDefault(config.DefaultCidVersion))
+		}
+
+		if !rbset && cfg.Import.UnixFSRawLeaves != config.Default {
+			rbset = true
+			rawblks = cfg.Import.UnixFSRawLeaves.WithDefault(config.DefaultUnixFSRawLeaves)
+		}
```

Comment:
:thought_balloon: Passing `--raw-leaves` produces CIDv1, which is ok I guess.

But asking for CIDv0 explicitly `--cid-version=0 --raw-leaves=true`  still produces CIDv1, which feels like a bug.  

Mind adding error handling for this scenario? (`error: unixfs raw leaves require cid version 1` should be enough)

Reply:
As per chatted on Slack, we're not doing this ATM. Fixing this opens a new can of worms which can be dealt with when actually changing the default CID version to 1.